### PR TITLE
feat: adding form label compoent

### PR DESCRIPTION
**Pull request by subhendukundu**
This PR will cover:
* Changed few files small indentations
* Added Label, which will be reused throughout all the form elements, also will be used for error and success etc messages in the form groups

Note: Please do give feedback where it can be written better way or reusable codes so that it can be added as a utility

### CODE REVIEW

File: fusion/Label.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    errorColor = '#FF0000',
+    successColor = '#00FF00',
+    warningColor = '#FFA500',
+    disabledColor = '#dddddd',
+    fontSize = '14px',
+    ...rest
+  } = props;
+
+  const thisClassName = className ? className : '';
+  const labelStyle = css`
```
Comment (vish288) [inline fusion/Label.js:24]: can these be moved out of the component def?

File: fusion/Label.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  const {
+    className,
+    disabled,
+    id,
+    required,
+    validationRequiredStateIndicator = '*',
+    validationState,
+    value,
+    color,
+    errorColor = '#FF0000',
```
Comment (vish288) [inline fusion/Label.js:15]: pass the default values of colors thru defaultprops.

File: fusion/Label.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  required: PropTypes.bool,
+  /**
+   *  The value to display next to the label if the field is required.
+   */
+  validationRequiredStateIndicator: PropTypes.string,
+  /**
+   *  The validation state of the assocat
+   */
+  validationState: PropTypes.oneOf(['', 'error', 'success', 'warning']),
+  value: PropTypes.string,
```
Comment (vish288) [inline fusion/Label.js:100]: comments for the rest of props?

**Review by vish288 — COMMENTED** [review]
if the comments are addressed, LGTM

**Comment by sbmadhav** [comment]
Please fix the conflicts

**Pull request closed without merging**